includes/templates/subsite/resources/git/hooks/pre-push

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env php
```

Comment:
What do you think about move this to the original location and set the binary path in our composer?